src/NServiceBus.Core/Faults/Forwarder/FaultManager.cs

```diff
@@ -82,29 +80,40 @@ void HandleProcessingAlwaysFailsForMessage(TransportMessage message, Exception e
 
             if (defer < TimeSpan.Zero)
             {
+                Logger.ErrorFormat(
+                    "SLR has failed to resolve the issue with message {0} and will be forwarded to the error queue at {1}",
+                    message.Id, ErrorQueue);
                 SendToErrorQueue(message, e);
                 return;
             }
-            sender.Send(message, new SendOptions(RetriesErrorQueue));
-
-            var retryAttempt = TransportMessageHeaderHelper.GetNumberOfRetries(message) + 1;
 
-            Logger.WarnFormat("{0} will be handed over to SLR for retry attempt {1}.", flrPart, retryAttempt);
-            busNotifications.Errors.InvokeMessageHasBeenSentToSecondLevelRetries(retryAttempt, message, e);
+            SendToRetriesQueue(message, e, flrPart);
         }
 
         void SendToErrorQueue(TransportMessage message, Exception exception)
         {
-            Logger.ErrorFormat(
-                "SLR has failed to resolve the issue with message {0} and will be forwarded to the error queue at {1}",
-                message.Id, ErrorQueue);
+            message.TimeToBeReceived = TimeSpan.MaxValue;
 
-            message.Headers.Remove(Headers.Retries);
+            if (message.Headers.ContainsKey(Headers.Retries))
```

Comment:
I think we could remove the FLR header here as well. It is added [here](https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.Core/Unicast/Transport/FirstLevelRetries.cs#L65)